xcode/Subconscious/Shared/Models/EntryStub.swift

```diff
@@ -18,8 +18,6 @@ struct EntryStub:
     let did: Did
     let address: Slashlink
     let excerpt: Subtext
-    // Is the excerpt shorter than the full body of the entry?
-    let isTruncated: Bool
```

Comment:
We no longer need to store this on the model if we truncate based on rendered size instead.

Reply:
I think we'd be better off determining if the text is truncated _in a given view_ which we can actually do with the gradient truncation helper I built - we can definitely explore a "show more..." on top of the gradient cutoff.